src/Elements/utilities.jl

```diff
@@ -0,0 +1,20 @@
+["Helper functions."]
+
+"""
+Find the value for ``key`` associated with a node's ``n`` configuration.
+
+Returns a ``Nullable{T}`` object. ``isnull`` must be called to determine whether
+an object was actually found or not.
+
+When the configuration of ``n`` does not contain the key ``key`` then all it's parents
+are searched in turn.
+"""
+function findconfig(n::Node, key::Symbol, T)
+    haskey(n.data, key) && return Nullable{T}(n.data[key])
+    # Stage 2: Parent's config.
+    while isdefined(n, :parent)
+        n = n.parent
+        haskey(n.data, key) && return Nullable{T}(n.data[key])
+    end
+    Nullable{T}()
+end
```

Comment:
Needs an explicit call to ``convert`` for the ``Nullable`` constructors. ``Nullable{T}(convert(T, n.data[key]))``. [Here](https://github.com/MichaelHatherly/Docile.jl/blob/de29d20a5404f690cbf5b97586c246898a0baee9/src/Cache/utilities.jl#L24) for example.